Configurations/CommonBase.xcconfig

```diff
@@ -23,11 +23,44 @@
 
 #include? "../../Internal/Configurations/HaveInternalSDK.xcconfig"
 #include? "<DEVELOPER_DIR>/AppleInternal/XcodeConfig/PlatformSupport.xcconfig"
-#include "SDKVariant.xcconfig"
+#include "Sanitizers.xcconfig"
+// SDKVariant.xcconfig is included via Sanitizers.xcconfig.
 #include "WebKitProjectPaths.xcconfig"
 #include "WebKitTargetConditionals.xcconfig"
 
-// Add $(WK_LIBCPP_ASSERTIONS_CFLAGS) to OTHER_CFLAGS in target xcconfig files.
+// Prefix Definitions
+//
+// WK_COMMON_ prefix is used to define variables in CommonBase.xcconfig
+// that may be used in project xcconfig files when $(inherited) is not
+// used.
+// Examples: WK_COMMON_OTHER_{CFLAGS,CPLUSPLUSFLAGS,LDFLAGS}.
+//
+// WK_DEFAULT_ prefix is used in Base.xcconfig project files for
+// single-value variables that may be overridden in CommonBase.xcconfig.
+// Example: WK_DEFAULT_GCC_OPTIMIZATION_LEVEL.
+//
+// WK_EXTRA_ prefix is used in Base.xcconfig project files for
+// multi-value variables that should be included in variable definitions
+// in CommonBase.xcconfig.
+// Example: WK_EXTRA_OTHER_{CFLAGS,CPLUSPLUSFLAGS,LDFLAGS}.
```

Comment:
Yes, that reads better.  I'll make the change.

Reply:
Actually, with what we found out last night, the `WK_EXTRA_` variables may not even be needed because we can just use `$(inherited)` and the same variable name in both files, so I think these will just go away.

Reply:
After doing the next patch refactoring, I think that it's best to keep `WK_COMMON_OTHER_CFLAGS` rather than shortening it to `WK_COMMON_CFLAGS`.

There are two reasons for this:
1. I basically don't have to do any research (or think very hard) after removing `WK_COMMON_` prefix to know which variable it's providing common values for.  I only have to look at the remaining variable name instead of looking at `Configurations/CommonBase.xcconfig`.
2. It would be inconsistent with `WK_COMMON_WARNING_CFLAGS` since they are both `CFLAGS` switches.  Why keep `WARNING_` but remove `OTHER_`?

So the next patch will remove `WK_EXTRA_` altogether, but keep `WK_COMMON_` as a prefix without altering the name of the variable it's providing common values for.